kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -247,6 +247,15 @@ export default {
     },
     kudosMessageTextLength() {
       return this.kudosMessageText && this.kudosMessageText.length;
+    },
+    fieldRequiredAlert() {
+      return this.kudosMessageText === '' && this.$t('exoplatform.kudos.warning.requiredField');
+    },
+    atLeastThreeWordsAlert() {
+      return this.kudosMessageText && (!(this.kudosMessageText.split(' ').length > 2) && this.$t('exoplatform.kudos.warning.atLeastThreeWords'));
```

Comment:
We can extract `this.kudosMessageText.split(' ').length > 2` to a computed variable (number of words)